src/arrayMethodSort.js

```diff
@@ -3,10 +3,34 @@
 /**
  * Implement method Sort
  */
+function compareStrings(previous, current) {
+  const previousString = String(previous);
+  const currentString = String(current);
+
+  return previousString > currentString;
```

Comment:
take a look to the doc, what sort return
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/sort